Versioned tests failing for pg-native@3.4.0

For `pg` and `pg-esm` versioned tests, the just-released `pg-native@3.4.0` fails.  The previous released version was `v3.3.0`, and [the diff is here](https://github.com/brianc/node-postgres/compare/pg-native%403.3.0...pg-native%403.4.0).

@brianc Running our versioned tests with (apparently) `pg@8.15.2` and `pg-native@3.4.1`, I'm getting the same error as we did previously: `Cannot find module './lib/build-result'`, which is not present in `pg`'s `lib` subdirectory.

> Running our versioned tests with (apparently) `pg@8.15.2` and `pg-native@3.4.1`, I'm getting the same error as we did previously: `Cannot find module './lib/build-result'`, which is not present in `pg`'s `lib` subdirectory.
> 
> Perhaps this should be a new issue on `pg`?

is there a way to reproduce this or create a self-contained test or steps I can replicate? I'm not sure exactly what's happening here...

--- a/test/versioned/pg-esm/package.json
+++ b/test/versioned/pg-esm/package.json
@@ -11,7 +11,7 @@
       },
       "dependencies": {
         "pg": ">=8.2.0",
-        "pg-native": ">=3.0.0 <3.4.0"
+        "pg-native": ">=3.0.0 <3.4.0 || >=3.4.5"
       },
       "files": [
         "force-native.test.mjs",

--- a/test/versioned/pg/package.json
+++ b/test/versioned/pg/package.json
@@ -10,7 +10,7 @@
       },
       "dependencies": {
         "pg": ">=8.2.0",
-        "pg-native": ">=3.0.0 <3.4.0"
+        "pg-native": ">=3.0.0 <3.4.0 || >=3.4.5"
       },
       "files": [
         "force-native.test.js",

